app/controllers/rescue_requests_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
   end
 
   def disaster_index
-    status_ids = [RequestStatus.find_by(name: 'Rescued').id, RequestStatus.find_by(name: 'Closed').id]
+    status_ids = [RequestStatus.find_by(name: 'Safe').id, RequestStatus.find_by(name: 'Closed').id]
```

Comment:
These can use the `[]` syntax to make them shorter: `RequestStatus['Safe'], RequestStatus['Closed']`.